applications/serial_lte_modem/src/slm_at_host.h

```diff
@@ -25,6 +25,13 @@ enum slm_datamode_operation_t {
 	DATAMODE_EXIT   /* Exit data mode */
 };
 
+/**@brief Exit modes in datamode. */
+enum slm_datamode_exit_t {
```

Comment:
No `_t` suffix please, it's not a typedef.

Reply:
updated, and sorry I always forget.